bentoml/utils/gcs.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Atalaya Tech, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from urllib.parse import urlparse
+
+logger = logging.getLogger(__name__)
+
+
+def is_gcs_url(url):
+    """
+    Check if url is an gs url
+    """
+    try:
+        return urlparse(url).scheme in ["gs"]
```

Comment:
Is `gs://` the standard way for Google cloud URI?  Could we document this, if you have a reference?

Reply:
@yubozhao Should we put the documentation link as a comment?

Here is the documentation for the same https://cloud.google.com/storage/docs/gsutil

Reply:
Ok great. Would you mind include that link in the form of comments for this funciton?